training.md

```diff
@@ -133,7 +133,7 @@ This section is divided into sort filters, sort of. See each section and find ou
 
 Why pathlib is fun:
 
-```py
+```
```

Comment:
```suggestion
```pycon
```
is the proper syntax highlight scheme for it. Also I suggest that we get rid of the python invocation and use this to show what platform we are running on:
```pycon
>>> import platform
>>> platform.system()
'Windows'
```

Reply:
Applied in both places, also replaced the invocation with a `platform` call. I used `Darwin` but feel free to change it to whatever.